server/db/migrate/20150505163032_remove_null_constraint_env

```diff
@@ -0,0 +1,5 @@
+class RemoveNullConstraintEnv < ActiveRecord::Migration
```

Comment:
I also have this change in my PR at: https://github.com/fusor/fusor/pull/98/files#diff-d4922f5b136f4ed24bd1c59c9f9acfa9R1